pyproject.toml

```diff
@@ -20,7 +20,8 @@ omit = [
     "venv/*",
     "fabfile.py",
     "*/__main__.py",
-    "tests/mock_web_api_server.py"
+    "tests/mock_web_api_server.py",
+    "workspace/teamdata/generate_report.py"
```

Comment:
I accidentally got the coverage tool to pick up this file (it wasn't previously), so it started complaining. It's not been changed and I'm expecting we'll be able to delete it soon, so I feel okay to ignore it for now.